src/functionalTest/groovy/com/autonomousapps/android/AbstractAndroidSpec.groovy

```diff
@@ -37,6 +39,7 @@ abstract class AbstractAndroidSpec extends AbstractFunctionalSpec {
     AGP_8_2,
     AGP_8_3,
     AGP_8_4,
+    AGP_8_5,
```

Comment:
Should we not add 8_6 to the test matrix?

Reply:
I wasn't sure, since the 8.6 branch is still in alpha.
I will add it to the list

Reply:
Done in https://github.com/autonomousapps/dependency-analysis-gradle-plugin/pull/1203/commits/0ff4dce1f78bcb4ffd00b7bb97506cd0cb3e0941